lib/modules/versioning/composer/index.ts

```diff
@@ -135,7 +149,14 @@ function isSingleVersion(input: string): boolean {
 }
 
 function isStable(version: string): boolean {
-  return !!(version && npm.isStable(composer2npm(version)));
+  if (version) {
```

Comment:
🤔 version should never null or undefined here.

Reply:
this wonders me too.. the previous implementation did `return !!(version && npm.isStable(composer2npm(version)));` with 2 exclamation marks, which suggests that `version` could be null. I was not able to pass 100% coverage without that test. What do you suggest in this case?

Reply:
I will remove that condition then... probably the previous implementation was not correct

Reply:
I would prefer to keep it as-is, just to be safe